accepted/0000-signed-packument.md

```diff
@@ -0,0 +1,70 @@
+# Signed Packuments
+
+## Summary
+
+Add support for validating packument signatures in the npm cli.
+
+## Motivation
+
+Today, there is no way to verify a packument's origin. This results in users unintentionally installing packages from the wrong source due to misconfigured npmrcs. This also allows caching layers, mirrors, and internal registries to tamper with files undetected.
+
+## Detailed Explanation
+
+We would like to give registries the ability to sign their packuments and give clients the ability to validate the bits they receive over the wire against those signatures.
+
+Packuments contain the integrity hashes of package tarballs. If a packument can be validated, the hash of the tarball is then trusted. This means signatures for tarballs are not necessary.
+
+## Rationale and Alternatives
+
+### [Notary](https://github.com/theupdateframework/notary)
+
+Notary is a project that allows anyone to have trust over arbitrary collections of data. This would allow us to validate the data uploaded to the npm registry against the _author's_ public keys. However, I believe, today packuments are generated by the registry meaning all packuments are generated by the same author.
+
+Notary may be an effective tool moving forward for validating that packages originated from a trusted/authorized author, irrespective of what registry a package was initially uploaded to, but that isn't the problem we are trying to solve here.
```

Comment:
To make sure this isn't forgotten, from the RFC meeting:

The major downside of this third party approach is that the install has already happened compromising your machine.  In this light it really needs to be a part of the install happening before the postinstall scripts.